src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -60,7 +60,9 @@ def stream_data(results, geojson):
     """
 
     sql = '''
-        SELECT sum(lengthkm) as lengthkm, stream_order
+        SELECT sum(lengthkm) as lengthkm,
+               stream_order,
+               avg(nullif(slope, -9998.0)) as avgslope
```

Comment:
Tagging @rajadain here to verify that this query matches the formula described here https://github.com/WikiWatershed/model-my-watershed/pull/2551#issuecomment-349994504

Reply:
Where does the `-9998.0` number come from? Can we make it a constant that is easier to read and conveys more context?

Reply:
It doesn't match. I'll be adding a fixup.

`-9998.0` comes from the original NHD table. I can set it to a constant so it's clear.

![screen shot 2017-12-07 at 10 55 43 am](https://user-images.githubusercontent.com/7633670/33724348-34b61f30-db3d-11e7-962a-60f0c18dcb0e.png)